packages/static-build/package.json

```diff
@@ -14,13 +14,24 @@
   },
   "scripts": {
     "build": "node build",
-    "test-unit": "jest --env node --verbose --runInBand --bail test/unit.test.js",
+    "test-unit": "jest --env node --verbose --bail test/build.test.ts test/prepare-cache.test.ts",
```

Comment:
Should we make `test-unit` grab all tests under a `unit` directory instead? I don't want to forget to add a new file to this list and wonder why my test wasn't being run.

Reply:
Perhaps, but it won't work currently cause of `integration.test.js` which shouldn't be run by `test-unit`.

Reply:
How about `*.unit.test` instead?